### PR TITLE
ULS settings for GP

### PR DESCRIPTION
**Pull request by ugochimobi**

### CODE REVIEW

File: LocalSettings.php [hunk 1]
```diff
@@ -4110,6 +4110,13 @@
 		'wmgUseTranslate' => false,
 		'wmgUseUniversalLanguageSelector' => false,
 	],
+	'wgULSIMEEnabled' => [
+		'default' => true,
+		'gratispaideiawiki' => false,
+	],
+	'wgULSWebfontsEnabled' => [
```
Comment (Reception123) [inline LocalSettings.php:4117]: setting the default only again...

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (Reception123) [reply LocalSettings.php:4117]: This isn't the default value: https://github.com/wikimedia/mediawiki-extensions-UniversalLanguageSelector/blob/master/extension.json#L101

**Review by Reception123 — CHANGES_REQUESTED** [review]

File: LocalSettings.php (hunk 1 above)
Comment (ugochimobi) [reply LocalSettings.php:4117]: Done

**Review by Reception123 — APPROVED** [review]

**Comment by Universal-Omega** [comment]
The default for `$wgULSWebfontsEnabled` is wrong here. It is false, meaning it doesn't even need to be set here...

**Comment by ugochimobi** [comment]
> The default for `$wgULSWebfontsEnabled` is wrong here. It is false, meaning it doesn't even need to be set here...

The default in extension.json is true? https://github.com/wikimedia/mediawiki-extensions-UniversalLanguageSelector/blob/master/extension.json#L101 
So it isn't wrong

**Comment by Universal-Omega** [comment]
> > The default for `$wgULSWebfontsEnabled` is wrong here. It is false, meaning it doesn't even need to be set here...
> 
> The default in extension.json is true? https://github.com/wikimedia/mediawiki-extensions-UniversalLanguageSelector/blob/master/extension.json#L101 So it isn't wrong

Sorry I had misread and was looking at `$wgULSMobileWebfontsEnabled`.

**Comment by ugochimobi** [comment]
I was only wondering though

**Pull request merged**